autospec/specfiles.py

```diff
@@ -1178,11 +1185,12 @@ def write_ruby_pattern(self):
 
     def write_cmake_pattern(self):
         """Write cmake pattern to spec file"""
-        self.subdir = "clr-build"
         if self.extra_make == "" or self.extra_cmake == " ":
             self.extra_make = "VERBOSE=1"
         self.write_prep()
         self.write_lang_c(export_epoch=True)
+        if self.subdir:
+            self._write_strip("pushd " + self.subdir)
```

Comment:
Hrm okay this makes sense to me but it also would make sense to me to replace the cmake_srcdir with subdir in the %cmake macro call below. I do think we should only need one of these however though. Thoughts?

Reply:
My understanding is that `cmake_srcdir` is relative to the clr-build directory (which is within `subdir` if set), so I think the code you're referring to is correct as-is